tests/solana.rs

```diff
@@ -854,7 +935,10 @@ impl<'a> SyscallObject<UserError> for SyscallLogData<'a> {
         memory_mapping: &mut MemoryMapping,
         result: &mut Result<u64, EbpfError<UserError>>,
     ) {
+        self.context.heap_verify();
+
         if let Ok(mut vm) = self.context.vm.try_borrow_mut() {
+            println!("sol_log_data");
```

Comment:
Is this for debugging?

Reply:
For testing purposes it is useful to know that `sol_log_data` is called. The mock solana (and substrate) print lots of stuff which helps debugging.

Reply:
Now it also logs the argument to `sol_log_data`